buildSrc/src/main/kotlin/Versions.kt

```diff
@@ -15,10 +15,5 @@
  */
 
 object Versions {
-    const val KOTLIN_VERSION = "1.5.21"
-    const val SPRING_VERSION = "5.2.13.RELEASE"
```

Comment:
Any reason not to use these constants any more?

Reply:
We are keeping the KOTLIN_VERSION  there since it's used in a plugin and the Koltin BOM reference. For the others it is easier to reason if they are defined with the dependency they affect. For most, all except Spring Boot, there is only one place such versions were referenced.

Reply:
Moved this part to a different PR.